src/Tests/Public/ResourceProviders/Microsoft.Network/DnsZone/New-ArmPrivateDnsZone.Tests.ps1

```diff
@@ -0,0 +1,133 @@
+$ScriptDir = Split-Path -parent $MyInvocation.MyCommand.Path
+Import-Module "$ScriptDir/../../../../../PoshArmDeployment" -Force
+
+InModuleScope PoshArmDeployment {
+    Describe "New-ArmPrivateDnsZone" {
+        
+        $ResourceType = "Microsoft.Network/privateDnsZones"
+        $ExpectedName = "Name1"
+
+        Context "Unit tests" {
+
+            $Depth = 1
```

Comment:
Are you sure a depth of 1 work? If you modify a sub properties does the test fail?

Reply:
Yes to both questions

Reply:
Yes to both questions